src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -835,3 +835,17 @@ export type Tag = {
 };
 
 export type Level = 'error' | 'fatal' | 'info' | 'warning' | 'sample';
+
+export type Meta = {
+  chunks: Array<Chunks>;
+  len: number;
+  rem: Array<Array<string | number>>;
+  err: Array<any>;
+};
+
+export type Chunks = {
```

Comment:
I think these need better names. `Meta` and `Chunk` feel like they could be absolutely anything, specifically because we export these types in our global types module.

Can we maybe stick some kind of prefix on these?